test/SparqlAntlrParserTest.cpp

```diff
@@ -619,16 +589,16 @@ TEST(SparqlParser, DataBlock) {
 }
 
 TEST(SparqlParser, InlineData) {
-  auto expectInlineData = makeExpectCompleteParser<parseInlineData>();
-  auto expectInlineDataFails = makeExpectParsingFails<parseInlineData>();
+  auto expectInlineData = ExpectCompleteParse<&Parser::inlineData>{};
+  auto expectInlineDataFails = makeExpectParsingFails<&Parser::inlineData>();
```

Comment:
This is off, they should have the same syntax, so possible `ExpectParsingFailse<&...>{};`

Reply:
Then we could also write functions that make us all the `Expect...` we need. So e.g. `auto expectInlineData, expectInlineDataFails = makeExpecters<&Parser::inlineData>();`